### DISCUSSION TITLE
Thank you for starting this!

### DISCUSSION THREAD

**Discussion by 1samhicks** [APIs-guru/openapi-directory#958] [General]
We need to a full list. Thank your getting this started!

**Comment by PleaseHelpBe** [comment]
Well you are most very welcome sir. I hope that anything I might be able to contribute towards the overall goal and resolution will be considered of benefit and appropriated responsibly.

**Reply by PleaseHelpBe** [reply]
What we are attempting to do is to sift through all of those variables, those elements that become confused within the paradigm of truth- mixed up between facts, assumptions suppositions, and general witness statements.

**Comment by MikeRalphson** [comment]
Not sure if this discussion is attracting AI bots or not, but I'm going to lock this now.
